tests/integration/targets/web_app_pool/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+# Cannot use win_feature to install IIS on Server 2008.
+# Run a brief check and skip hosts that don't support
+# that operation
+- name: check if win_feature will work on test host
```

Comment:
We can remove this check and the block as we now have a minimum versions of 2016 so `Get-WindowsFeature` is always present.

Basically just start with line 11 without the `when: module_available.rc == 0` check right at the end.

Reply:
There is an always statement to make sure the test pool is deleted so maybe worth keeping the block just for this case (without the when)

Reply:
Yep sorry I got ahead of myself, I meant remove just the when.